lib/App/DuckPAN.pm

```diff
@@ -268,21 +270,31 @@ sub execute {
 	my @arr_args = grep { $_ !~ /^-/} @{$args}; # Command line switches make it here, so we try to remove
 	App::DuckPAN::Cmd::Help->run(1) if scalar @arr_args == 0;
 	if (@arr_args) {
-		my @modules;
-		my @left_args;
+		my (@modules, @left_args, $ddg);
 		for (@arr_args) {
-			if ($_ =~ /^www/i ||
-				$_ =~ /^dist/i ||
-				$_ =~ /^ddg/i ||
-				$_ =~ /^app/i) {
+			if (/^www/i ||
+				/^dist/i ||
+				/^(ddg)$/i ||
+				/(opensourceduckduckgo)$/i ||
```

Comment:
wondering if we should make aliases for the ia bundles so we don’t have to `duckpan DDG::SpiceBundle::OpenSourceDuckDuckGo`?

e.g. `duckpan Spice` does `duckpan DDG::SpiceBundle::OpenSourceDuckDuckGo`